app/models/discourse_reports/newsletter.rb

```diff
@@ -25,6 +25,11 @@ def pdf
       kit.to_pdf
     end
 
+    def img
+      kit = IMGKit.new(File.new(html), :quality => 50)
+      kit.to_img
```

Comment:
I tried testing with a really large HTML file and the image came out black towards the end of the file.